ignite/pkg/gomodule/gomodule.go

```diff
@@ -68,9 +68,11 @@ func ResolveDependencies(f *modfile.File) ([]module.Version, error) {
 	}
 
 	for _, req := range f.Require {
-		if req.Indirect {
-			continue
-		}
+		/*	Commented out to include indirect dependencies as proto files may import from them
+			if req.Indirect {
+				continue
+			}
+		*/
```

Comment:
Right now I'm not sure if this would create an issue with the current implementation so maybe in this case I would keep the existing behaviour as default to avoid having any side effect in the existing implementation by adding a boolean argument like for example:

```go
func ResolveDependencies(f *modfile.File, includeIndirect bool) ([]module.Version, error)
```

Ideally we should confirm that the removal of the `if res.Indirect` condition won't have any side effects.